review skema data base

NurindahYuniarti2A opened the pull request:
tapi belum beres..

NurindahYuniarti2A committed:
review skema data base

tapi belum beres..

nmuntaaza committed:
Memperbaiki penamaan relationship

Memperbaiki penamaan relationship pada "CDM_Travlendar_versi2"

Dianinsani123 commented:
btw mana ini db yang bentuk pwd nya ? ga ngerti.. cuma ada CDM_Travlendar_versi2 itu pun (dot)cdm

Dianinsani123 commented:
eh maaf maaf, bisa ternyata.. hehe

Dianinsani123 commented:
![1](https://user-images.githubusercontent.com/29325875/31282602-43a70ab2-aadd-11e7-9875-412c26edcfed.png)
1. kenapa ada entitas yang tidak memiliki hubungan dengan entitas lainnya ? 
2. sejauh ini untuk kecamatan, kelurahan dan jalan tidak dibutuhkan, karena dalam data kasus tidak ada, yang ada hanya kota dan provinsi saja. bangka -> jakarta ; tanggerang -> jakarta.
3. untuk atribut waktu keberangkatan dan tiba memang akan dihilangkan atau bagaimana ? menurut saya, untuk waktu keberangkatan dan tiba tetap diadakan namun diluar dari entitas event itu sendiri.

![2](https://user-images.githubusercontent.com/29325875/31282699-8ef58aca-aadd-11e7-853e-88124a1efd45.png)
1. berikan penamaan yang jelas dan mudah dipahami pada setiap hubungan antar entitasnya. saya sendiri bingung memahami antara entitas tranport estimation time dan jarak.

NurindahYuniarti2A committed:
sql script travlendar update 2

ketika di generate tampil table result , tapi disitu ada 4 category table index yang sepertinya ada kesalahan.. tapi script tetap ada.. apakah itu akan berpengaruh di database nya tau bagaimana?
